src/settings/_transitions.scss

```diff
@@ -0,0 +1 @@
+$trs-default: 0.3s ease !default;
```

Comment:
We have the `_motions.scss` file to the animation time values

👉🏼 https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_motion.scss

I think that we can have this var inside of `_motions.scss`

Reply:
Done 👍 I didn't notice that file since it's still not used in our theme. Thanks!